packages/nodes-base/utils/sendAndWait/utils.ts

```diff
@@ -480,8 +486,6 @@ export function getSendAndWaitConfig(context: IExecuteFunctions): SendAndWaitCon
 		.replace(/\\n/g, '\n')
 		.replace(/<br>/g, '\n');
 	const subject = escapeHtml(context.getNodeParameter('subject', 0, '') as string);
-	const resumeUrl = context.evaluateExpression('{{ $execution?.resumeUrl }}', 0) as string;
```

Comment:
Does this mean that `$execution.resumeUrl` will no longer work because it would be missing the signature?

Reply:
yes, unfortunately, this is breaking change for responses for executions started before update, still looking if it's possible to avoid

Reply:
But wouldn't it also be breaking for new executions, if your workflow uses `$execution.resumeUrl` in an expression?